schema.graphql

```diff
@@ -543,12 +543,51 @@ type NftUpdate @entity {
   tx: String!
 }
 
-
 type Template @entity{
   id: ID!
   fixedRateTemplates: [String!]
   dispenserTemplates: [String!]
   ssTemplates: [String!]
 }
 
+# Not tracking allocationToId or idToAllocation
+# id = {user}
+type veAllocateUser @entity{
+  id: ID!
```

Comment:
minor suggestion, write the comment above the prop so the users see the comments in the schema like: 
```
"id = {user}"
id: ID!
```

If the comments were just provisional for internal use, ignore this :)